### DISCUSSION TITLE
Make Github Copilot not override intellisense.

### DISCUSSION THREAD

**Discussion by Xyphyn** [community/community#9650] [Copilot Conversations]
I wish there was an option to make it so that when github copilot has a suggestion, it doesn't override intellisense. Normally it makes it disappear, and this is really annoying.

**Comment by sonic1981** [comment]
Plenty of people already mentioned this. Prob upvote one of those. https://github.com/github/feedback/discussions/9430

**Reply by zenerbufen** [reply]
Sonic, you linked to a discussion about the issue with VS CODE, and the copilot extension FOR vs CODE that has 'already been fixed' 

Intellisence & intellicode currently still get overwritten / disabled / conflicting keybindings in Visual Studio (NOT CODE) 2022 with Visual Studio Co Pilot extension (which is a separate program, maintained by a separate team) 

Github Copilot still overwrites, and blocks intellisense in VS 2022, and your link is not applicable to that, as it is marked fixed / closed & applies to a different product.
